pkgs/development/python-modules/fakeredis/default.nix

```diff
@@ -19,11 +20,13 @@ buildPythonPackage rec {
   version = "1.7.0";
   format = "setuptools";
 
-  disabled = pythonOlder "3.5";
+  disabled = pythonOlder "3.7";
```

Comment:
Is there a reason that this was changed?

Reply:
`unstable` doesn't support Python 3.6 or 3.5 anymore out-of-the-box thus I can only check if the package build locally on Python 3.7 and above. Also, 3.6 is EOL.